lldb/source/Plugins/Language/CPlusPlus/LibCxxUnorderedMap.cpp

```diff
@@ -162,10 +162,21 @@ lldb::ValueObjectSP lldb_private::formatters::
       if (!node_sp || error.Fail())
           return nullptr;
 
-      value_sp = node_sp->GetChildMemberWithName("__value_");
       hash_sp = node_sp->GetChildMemberWithName("__hash_");
-      if (!value_sp || !hash_sp)
+      if (!hash_sp)
         return nullptr;
+
+      value_sp = node_sp->GetChildMemberWithName("__value_");
+      if (!value_sp) {
+        // Newer libc++ versions wrap the `__value_` in an anonymous union.
```

Comment:
It might be useful to specify when this was introduced in libc++ in case this logic gets updated again in the future for an even newer libc++.

Reply:
Agreed

Let me know if latest comment provides enough context/info

Reply:
Definitely better. I assume phabricator will be read-only mode for quite a while to come, so I'm not too worried about it being difficult to find. We should probably update it with a commit hash once it lands though.

Reply:
Added the commit hash